Directory.Build.props

```diff
@@ -13,6 +13,10 @@
     <NoWarn>NU1507;NU1903;CS9107</NoWarn>
     <BuildTransitivePath>buildTransitive/$(TargetFramework)/</BuildTransitivePath>
     <BuildPath>build/netstandard2.0</BuildPath>
+
+    <SignAssembly>true</SignAssembly>
```

Comment:
If this is in the targets file instead, you could do something like:

`<SignAssembly>'$(IsLibraryPackage)' == 'true'</SignAssembly>`

That variable is applied to the NuGet package projects, and everything else should remain unsigned.

Reply:
At least as of when the PR was new, _some_ non-shipping projects _did_ need to be signed because you had InternalsVisibleTo exposed to them. And as of yesterday when I refreshed the PR, I saw you still have IVT in use, though I didn't pay attention to which assemblies you exposed internals to. 

If you feel confident that you don't use IVT from shipping to non-shipping assemblies, then yes, I can give your suggestion a try.